UI_UX/Frontend/src/components/AccountInfo.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Loading } from 'carbon-components-react';
+import { Address } from './Address';
+
+export const AccountInfo = ({
+  loadingBalance,
+  loadingAccountInfo,
+  myBitBalance,
+  ethBalance,
+  address
+}) => {
+  let loadingIcon = undefined;
+  if (loadingBalance)
+    loadingIcon = (
+      <Loading
+        className="AppHeader_balance_loading"

```

Comment:
Please use BEM. Classname should be `AppHeader__balance--is-loading`

Reply:
fixed